qulice-checkstyle/src/test/resources/com/qulice/checkstyle/AnnotationIndentation.java

```diff
@@ -7,12 +7,10 @@
  * Simple.
  * @since 1.0
  */
-@SuppressWarnings(
```

Comment:
This change shouldn't be needed, original code already looks well-indented.

As far as I understand, this is a bug in checkstyle (`https://github.com/checkstyle/checkstyle/pull/10093`), and it is fixed in version 9.0